content/blog/2016/2016-08-24-jenkins-world-speaker-blog-flawcheck.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Jenkins World Speaker Highlight: Secure Container Development Pipelines with Jenkins"
```

Comment:
Again, these titles should not have the prefix "Jenkins World Speaker Highlight"